cpp/frontend_pyreadline.h

```diff
@@ -43,18 +43,23 @@ class Readline {
   static constexpr uint16_t field_mask() {
     return maskbit(offsetof(Readline, completer_delims_)) |
            maskbit(offsetof(Readline, completer_)) |
-           maskbit(offsetof(Readline, display_));
+           maskbit(offsetof(Readline, display_)) |
+           maskbit(offsetof(Readline, latest_line_));
   }
 
   int begidx_;
   int endidx_;
   Str* completer_delims_;
   completion::ReadlineCallback* completer_;
   comp_ui::_IDisplay* display_;
+  Str* latest_line_;
```

Comment:
This could use some comments about what the state means, i.e. latest_line_ != nullptr vs. ready_ vs EOF, etc.